site/blog/2022-10-27-clean-your-aws-account.mdx

```diff
@@ -0,0 +1,32 @@
+---
+slug: clean-your-aws-account
+title: Clean your AWS account just by clicking a button
+authors: [depombo, dfellis, aguillenv, yrobla]
+---
+
+# Clean your AWS account by simply clicking a button
+
+One of the main complexities of working with AWS is the amount of resources that can be created, either by manual or automated workflows. The chained resources created and dependencies can quickly get out of control, and it could be a nightmare to clean up then.
```

Comment:
Nit

```suggestion
One of the main complexities of working with AWS is the amount of resources that can be created, either by manual or automated workflows. The chained resources created and their dependencies can quickly get out of control, and it could be a nightmare to clean them up.
```